components/script/task_queue.rs

```diff
@@ -110,12 +110,12 @@ impl<T: QueuedTaskConversion> TaskQueue<T> {
 
     /// Reset the queue for a new iteration of the event-loop,
     /// returning the port about whose readiness we want to be notified.
-    pub fn select(&self) -> &base_channel::Receiver<T> {
+    pub fn select(&self) -> &crossbeam_channel::Receiver<T> {
```

Comment:
Methods `recv` and `try_recv` below this `select` method used to return `Result<T, ()>`. Perhaps we should restore that.